cudaaligner/tests/cudaaligner_test_cases.cpp

```diff
@@ -88,6 +88,10 @@ std::vector<TestCaseData> create_cudaaligner_test_cases()
GATGCCACGCATAGACGACGTGTCTTGCGTGGTAGCAACGACCTCTTCGATACTTCATCGCGGTAAATTCTGTACGGGATCCTCACACCAAGGTCTAACGCATTACGCGTCGGTTTCGGCCAGACTGGCGCTACATCGTTCATCATCAACAGATGCGCCAGCGATTTATGACAATTGCGGTCGATCATGGCGTACTGGTGGCGCGGCGTACACCCACAATTTTGTTCGACGATCGATGTTCGTTGGTAGCAATGATATAATAACTCTGTTCCGCGCCAAAGTCGCGGTGTACTCTTCCGCGAACTTCGGTGTACAGGTGTGGTCAGAGCAACGAACGTGACGCCATTGACCGAAATAGAGACATCAGCCCATGAAGTATATTCCCGCCGAAAAAAATCATAAAACAGACAGCCAAGCCAGGCTTTTTGACAGCCGCGGTGCCGCTATGCCTGAAACTACCAAAAGTGTACTTCGCTCTTGACGTAGGTAAACAAAACTTTTGACCGAACGGCGGTGTGGTATGAAGATACGTCGGTGTACTGACATACAGGCTGGCGATATCTTCGCTGCTGGCGCATATTCAAAAAACCAGAGCGCCATCATATCCTGCACGCTGACATCCATCGTCGAGTGGGTGTTGATAAGAAAGGCATAAAGCGGGAATGTTGTTGGCGATTGTTATCGCTACATAAATCGAGACTGTACTCATCCCCGGTCAAAAATCACGCCGCAAATTCGAGGGTTATGCTCGATAAATTTCAACAAATCAACATGTTTTGTGGCCAGATAATCTGACGCCCTCGCGCCGACTCCAGTTCTTTGATGGGCTCATCTTTATAAAAGACGCCATGCGGTCCCATAATGGCAATGATGTTCATGTGTTCCTCCTGGAAAATCCTTCCTTGTCTGTATATCCGACCATTGGCGATAAAAAGACCACCGAGAAGTGACCTTTTTCAGGGCTTTTACCAATTACGCGTAACCGTAGCTCATCAGGCGCTGATAACGACGATTTTTTAAATCTTCAGTGCTTAACACGTCATGAGATCGGCCAGATCACCGCCAGCAGTTGCTCAGCGATACGCCGCCATCGGCTTCCGGGTTACGGTGAGCACCCTTTGGTGGTTCCGGGATGATGGAGTCGATCCGGTTCCAGTTCCTTTCAGACGCGGAACAATGAGTACCCATCGCTTCAGCCCCACTTCGGCGGCGCTTTGTCGGCGCTCTTCCACAGAATCAGACACAACGCTTCCGGCGGAATAACGGAATAGGTGCTGTATTTGCAGCATCATATTCACTTTATCGCCCACGCCAATCGCCAGCGCACTACCGCCAGAACCACGCTTCACCGATAACCGTACAAACTACCGGTACGCCGAGGCGAGACATTTCCGCGGGTTGCATTAAATGGCTCAGACTGACCACGCTCTTCTGCGCCCACGCCAGATAAGCCCGGTCGATAAGGTGATGATAAGCATCGCCAGCGTTCAGCCATTTGCTGTCAGACGCAGTGCTTTTGCGGTAACCTTCTGGCGCTGGCGCACTGCCAAGTTACGTTTTTCTTTTTGGTTTCACGACCTTTTTGACGACCAATGATCATCACCAGGACGACTCATCAGGTGGGCGATACCACCGACGATGTTCTCGCAATCTTCGCCCTGCTGGTCGCCAGCCAGTTCGTCAAATTCATCAAATCGCCAGGCAGACGCGTAATCCGAGGTATAAGGACACTGTGGATGGCGTACCAGTTGCGCAATCTGCCATGCCAGATCGGCGAAAGATTTTACGTAATCAGTTCTACGCTTTTTCACGCGAACGATGCGCTTCTTCATCGATGTTAATATCCAGTTTCTCATCCTGACGGCTAACCGCAGTCAGAGAATCGATTTTCAACTACAACTCTGCAATCAGCTGCAGTGCAAATCAAGGAAATTCCAGACTCACTAAAAATGTTCTGTATTGATAAACTCCGGATTCACCTGCTCCAGGGCTGGCTAAAAATTACCAGGATCGTTGTCGGCGATCGCTCGGAAGACACGCCACGTCGCGCCAAAACGCAACCGCGCGCTGGCATTAATCGCCTCTGATGATGAGATAACTTTGCTGATTGTCCAGAGCGGTGGGGTTTCCGGGGCAGCCAGTCAGGTTTACCAAAAGCTGGTCATCAATTTGCCTGTCGTCCGTGATTACATAAAAGCCCGCGAGCATATTTCAGGCTTCGTCAATATCCATCACTTCGCCGACAGTTATTCAGCCGCTATGATCATCGGCTGACCTGTCCACACGACGTCGGTATCGGTCTTTACCAGCAGTTTTCGAGTGGCCCAACATCAGGGCGTCAGTAAATAACATCTTCCAGCCACCGGAACGGTCATCAGCGTGCAGATACCGATACGATTGCCGCGGTGACCTGAACCGCGGCAACAACGAGCCCCACCGGCGTGACTTTACGTTCTGTCGTTGCGTATCGCGCCGACGCCGCGCCTCCGACATAACGCTCAATCTCTTTTAAATACTGGTTGATAAGTCGCCACGATCTGGGAATTTCAGGCTTCAACGTTCCGTTCCCCATCTGATACCGCCTGCTCTATGCACGGTTGGCAACTGGCGTGGGTCTTCCGTTGTTCAGCTCGCTGAAATGACACGCCGAACATATCGGCCTGACCGACGTGGCTTCCACCTTTCGCGTGTTGATCTGCCGCTTTTAACATCATAGCGAGTTCATCAGCGCGCTGCGCGATGTGTTGAGCAGATAAACACCCCGGACGCGATCAGTTTTTCCAGCACGCGACAGTTCTCGCTTTGCAGTGGTGTCCAGTTACCGGCGCAGAGAGATCGTCGGATCGCCAAGATGGCCATACCATTACCGGAGAAACTGATGATGGCCTCAATCAGGACCTTCACCGACCTTTTGATCAACACCAATACCATACCACGATTCGCCGTCATTGACGTGAAATATTAAGGCGGAGTTTATATCTGGTGGCAGGATTTTCAG";
```

Comment:
Optional: save this in a file like it's done in `cudamapper/data`?

Reply:
I think this is definitely a good idea - especially the last test strings are too long to hard code them.
However, I'd like to postpone this change, since this is mostly a bug fix PR.

Reply:
opened #505

Reply:
sounds good to me